doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,233 @@
 
-// Define the problem and tell reader what he/she will learn from this guide.
+== 5) Create EC2 Instances
 
-== Prerequisites
+Let's start creating our instances via `LaunchInstanceWizard` under `EC2 > Launch Instances` on AWS Console.
 
-// Define the prerequisites for the guide.
+* Choose an Amazon Machine Image (AMI). `Amazon Linux` is used in this guide:
 
-////
-Write the steps in your guide here by making each step a different section
-////
+image::ss5.1-create-instance.png[]
+
+
+* Choose an instance type:
+
+image::ss5.2-create-instance.png[]
+
+
+* Now configure instance details with the *_VPC_, _subnet_ and _IAM roles_* you created above. Notice that the number
+of instances is 2. Also, enable `Auto-assign Public IP` to establish SSH connections later on.
+
+image::ss5.3-create-instance.png[]
+
+
+* Next, add a unique tag to the instances. This is optional but recommended if your AWS account has many running
+instances associated with. We will configure Hazelcast to filter discovered EC2 instances based on this tag:
+
+image::ss5.4-create-instance.png[]
+
+
+* Finally, select the security group you created above. You also can use different security groups for different
+instances as long as you do not set `security-group-name` property in Hazelcast AWS configuration (more on configuration
+details in the next section):
+
+image::ss5.5-create-instance.png[]
+
+As the last step, select your https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair]
+for the instances and that's it. You can launch instances now.
+
+
+== 6) Create a Hazelcast Cluster
+
+* Now that you have 2 instances running with the same IAM role, let's connect to each of them via SSH:
+
+[source, shell]
+----
+ssh -i "<your-key-pair>.pem" ec2-user@<instance-public-ip>
+----
+
+[NOTE]
+====
+If you encounter any problem regarding the SSH connection, see
+https://aws.amazon.com/tr/premiumsupport/knowledge-center/ec2-linux-ssh-troubleshooting/[SSH troubleshooting page]
+in AWS documentation.
+====
+
+* After SSH connection is established, install JDK8 (or any newer version) to the instances:
+
+[source, shell]
+----
+sudo yum -y update && sudo yum install java-1.8.0-openjdk
+----
+
+* Verify that it is installed:
+
+[source, shell]
+----
+java -version
+----
+
+* Now, let's install Hazelcast JAR. Do not forget to set *HZ_VERSION* variable or change the URL with the version you use:
+
+[source, shell]
+----
+JAR_URL="https://repo1.maven.org/maven2/com/hazelcast/hazelcast-all/${HZ_VERSION}/hazelcast-all-${HZ_VERSION}.jar"
+curl -sf -O -L $JAR_URL
+----
+
+[NOTE]
+====
+To make instances find each other and form a cluster, {aws-plugin-url}[Hazelcast AWS plugin]
+must be used. This is included in `hazelcast-all` and you are all set if you downloaded from the URL above. Otherwise,
+you need to include `hazelcast-aws` explicitly.
+====
+
+* Create a Hazelcast configuration now. A minimal configuration with AWS discovery enabled will look like the yaml
```

Comment:
How about first just starting `hz start` and showing that it works. And then adding this filtering by tag. Also maybe better to try to use config override than providing the whole configuration?